Improve Prometheus documentation

philipnrmn opened the pull request:
## Description
<!-- Link to JIRA issue -->
[DCOS-22016](https://jira.mesosphere.com/browse/DCOS-22016) - Update prometheus documentation

## Urgency
- [ ] Blocker <!-- Ping @sascala, @stbof, or @pavisandhu for review -->
- [x] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

cneth commented:
+1 LGTM.  

Thanks for cleaning this up!

cneth commented on pages/1.10/metrics/prometheus/index.md:
```diff
@@ -3,51 +3,80 @@ layout: layout.pug
 
 1.  On every master node:
-    1. Download the master [systemd service file](https://raw.githubusercontent.com/dcos/dcos-metrics/master/plugins/prometheus/systemd/dcos-metrics-prometheus-master.service) to `/etc/systemd/system`.
+    1. Download the plugin systemd service file from downloads.mesosphere.io: [prometheus-plugin.service](https://downloads.mesosphere.io/dcos-metrics/plugins/prometheus.service)
```
The unit file at that location contains this line:
`ExecStart=/opt/mesosphere/bin/dcos-metrics- --dcos-role master --prometheus-port 61091`

Notice that 'prometheus' is missing from the executable name...  You mention the correct name below but it seems like it would be better if the file you downloaded had the correct name to start with.

anithamuthu79 replied:
@philipnrmn 
You need to add a "period" after  the below line
 1. Download the latest Prometheus plugin binary from downloads.mesosphere.io: [prometheus-plugin](https://downloads.mesosphere.io/dcos-metrics/plugins/prometheus)

anithamuthu79 replied:
@philipnrmn in line number 34 and 41, you can rewrite this 
"On every master node:" as "On every master node do the following steps:"
"On every agent node:" as "On every agent node do the following steps:"

Line 59: Rewrite 
There are many ways to run a Prometheus server. This is the simplest way to get started with self-hosted metrics on DC/OS. 
as
" There are several ways to run a Prometheus server. The simplest way to get started with self-hosted metrics on DC/OS is listed below:"

cneth commented on pages/1.10/metrics/prometheus/index.md:
```diff
@@ -3,51 +3,80 @@ layout: layout.pug
+        Description=DC/OS Metrics Prometheus Plugin
+
+        [Service]
+        ExecStart=/opt/mesosphere/bin/dcos-metrics-prometheus -dcos-role master
```
Should the port be mentioned here?   `--prometheus-port 61091`?

philipnrmn replied:
61091 is the default, and it's also expected by the prometheus as configured in this guide. Changing it will break the service discovery mechanism. I'm happy to add it to the ExecStart line.

philipnrmn commented:
@cneth addressed your comments; note that downloads.mesosphere.io has heavy caching so the file doesn't appear to have changed yet but will in a few hours. It now looks like [this](https://raw.githubusercontent.com/dcos/dcos-metrics/master/docs/resources/prometheus.service)
